src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingWareHouse.java

```diff
@@ -145,6 +139,13 @@ public boolean registerWithWareHouse(final IBuildingDeliveryman buildingWorker)
         return true;
     }
 
+    @Override
+    public void unregisterFromWareHouse(final IBuildingDeliveryman buildingWorker)
+    {
+        final Vector3d vec = new Vector3d(buildingWorker.getID().getX(), buildingWorker.getID().getY(), buildingWorker.getID().getZ());
```

Comment:
It seems like we're converting to Vector3d in a lot of cases. Should we add a getVec3d to buildingworker?

Reply:
Hmm, would be an overkill, since there is a new Vec3d(blockpos) constructor as well